react/features/video-menu/components/web/RemoteVideoMenuTriggerButton.js

```diff
@@ -343,26 +330,43 @@ class RemoteVideoMenuTriggerButton extends Component<Props> {
         );
 
         if (isMobileBrowser()) {
-            buttons.push(
+            actions.push(
                 <ConnectionStatusButton
                     key = 'conn-status'
                     participantId = { participantID } />
             );
         }
 
         if (showVolumeSlider) {
-            buttons.push(
+            actions.push(
                 <VolumeSlider
                     initialValue = { initialVolumeValue }
                     key = 'volume-slider'
                     onChange = { onVolumeChange } />
             );
         }
 
-        if (buttons.length > 0) {
+        if (buttons.length > 0 || actions.length > 0) {
```

Comment:
isn't this altering the current ordering?

Reply:
No, because the so named `actions` are displayed only on mobile and at the end of the list. The buttons are kept in the same order as to when they were pushed.